pkg/providers/aws/provider_redis.go

```diff
@@ -741,6 +741,10 @@ func buildElasticacheUpdateStrategy(ec2Client ec2iface.EC2API, elasticacheConfig
 			modifyInput.SnapshotWindow = elasticacheConfig.SnapshotWindow
 			updateFound = true
 		}
+
+		if applyImmediately {
```

Comment:
Does this only set it to true? If it were false line 745 would not enter this if, right? Is that OK?

Reply:
Good point at some time in the future we may want to set this to false on aws